Arrowgene.Ddon.GameServer/Handler/QuestGetRewardBoxHandler.cs

```diff
@@ -40,7 +40,7 @@ public override void Handle(GameClient client, IPacket packet)
                 if (quest == null)
                 {
                     Logger.Error($"Failed to find quest for {boxReward.QuestId}. Unable to retrieve rewards.");
-                    continue;
+                    throw new QuestDoesNotExistException(boxReward.QuestId);
```

Comment:
Extend GameRequestPacketHandler so it catches this exception and returns an error response packet